app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -257,6 +257,11 @@ def combine_alert_data(all_alerts, alert_name)
             new_alert_data[:name] = value if key.end_with?('_AlertName')
             new_alert_data[:result] = value if key.end_with?('_AuthenticationResult')
             new_alert_data[:region] = value if key.end_with?('_Regions')
+            new_alert_data[:disposition] = value if key.end_with?('_Disposition')
+            new_alert_data[:model] = value if key.end_with?('_Model')
+            if key.end_with?('Regions_Reference')
```

Comment:
👋🏿 @dawei-nava  - is there value here in collecting the Regions_Reference IDs?

Reply:
Yes, but the id self is not kept since it does not provide much information outside of the response itself, the informaiton collected is the region and which side of the region located on the ID.

Reply:
@dawei-nava  ...  this now allows us to track the side of the ID the region is located 👍🏿 
can we add some tests for this new region_ref alert data?

Reply:
@amirbey, added.